app/models/view/reports/Report.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2019 Dropbox, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package models.view.reports;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSetMultimap;
+import models.internal.impl.DefaultReport;
+import models.view.scheduling.Schedule;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+/**
+ * View model for a {@link models.internal.reports.Report}.
+ *
+ * Play view models are mutable.
+ *
+ * @author Christian Briones (cbriones at dropbox dot com)
+ */
+public final class Report {
+    private Report() {
```

Comment:
The other view model constructors are not private. This will probably make testing harder, and Jackson is going to call the private ctor anyway. If you want this pattern, we should discuss at applying it across the board.